storage/inspector/src/main.rs

```diff
@@ -120,6 +124,36 @@ fn list_txns(db: &LibraDB) {
     }
 }
 
+fn list_accounts(db: &LibraDB) {
+    let version = db
+        .get_latest_version()
+        .expect("Unable to get latest version");
+    let backup = db.get_backup_handler();
+    let iter = backup
+        .get_account_iter(version)
+        .expect("Unagle to get account iter");
+    for v in iter {
+        match v {
+            Ok(x) => {
+                let accs = AccountState::try_from(&x.1).expect("Failed to read AccountState");
+                let addr = accs
+                    .get_account_address()
+                    .expect("Could not get address from state");
+                match addr {
+                    Some(x) => println!("Address: {:?}", x),
+                    None => println!("Invalid Address: {:?}", addr),
+                }
+            }
+            Err(x) => println!("Got err iterating through AccountStateBlobs {:?}", x),
+        }
+    }
+    let iter = backup
+        .get_account_iter(version)
+        .expect("Unagle to get account iter");
+    let num_account_state = iter.count();
```

Comment:
this would  iterate the database again, ideally you should just count it above to save some effort.